prow/config.yaml

```diff
@@ -6914,20 +6914,38 @@ periodics:
       - --timeout=40
       - --root=/go/src
       env:
+      # This job runs k8s integration test thus need dind
+      - name: DOCKER_IN_DOCKER_ENABLED
+        value: true
       - name: GOOGLE_APPLICATION_CREDENTIALS
         value: /etc/service-account/service-account.json
       - name: USER
         value: prow
       - name: GOPATH
         value: /go
+      # docker-in-docker needs privileged mode
+      securityContext:
+        privileged: true
       volumeMounts:
       - name: service
         mountPath: /etc/service-account
         readOnly: true
+      - name: docker-graph
+        mountPath: /docker-graph
+      - name: var-lib-docker
```

Comment:
this volume is unnecessary

Reply:
do we want to clean it up from other jobs? (I was copy pasting)

Reply:
probably yeah, thought I thought only canaries were still doing this. Leave the `*-prow` jobs as-is.

Reply:
done